nimoy/orb.yml

```diff
@@ -3,7 +3,7 @@
 version: 2.1
 
 orbs:
-  aws-ecr: circleci/aws-ecr@6.12.2
+  aws-ecr: circleci/aws-ecr@8.1.2
```

Comment:
❌ this does not seem to be the nimoy orb in the orbs repo. the current one was built by orb2.ym